test/examples/nginx/nginx.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.16.1
```

Comment:
@chetan-rns I think OpenShift won't allow root image. So the containers image path would be ```nginxinc/nginx-unprivileged:1.16.1```

Reply:
Holding this pr till the review comment is not addressed 
/hold

Reply:
Tried nginx:1.16.1 and it was deployed successfully. I've updated the image any way